runtime/gc_check/CheckCycle.cpp

```diff
@@ -429,6 +429,13 @@ GC_CheckCycle::initialize(const char *options)
 void
 GC_CheckCycle::run(GCCheckInvokedBy invokedBy, UDATA filterFlags)
 {
+	UDATA originalMiscFlags = _miscFlags;
+	/* set the midscavenge miscFlags if is in concurrent scavenger to check forwarded pointers */
```

Comment:
Please make comment more clear, try to explain higher level motivation for the change.
For example:
```
/ * Enable recognition of Forwarded Pointers can be discovered during 
  * Concurrent Copy phase of Concurrent Scavenge.
  * Also suppress reporting of discovered Forwarded Pointers.
  */
```